deployments/helm/nvidia-dra-driver-gpu/templates/controller.yaml

```diff
@@ -25,7 +25,7 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      {{- include "nvidia-dra-driver-gpu.selectorLabels" . | nindent 6 }}
+      {{- include "nvidia-dra-driver-gpu.selectorLabels" (dict "context" . "componentName" "controller") | nindent 6 }}
```

Comment:
By the way, I have now learned that this left-chomping-nindent-technique is the holy grail of Helm templates: https://github.com/helm/helm/issues/3854#issuecomment-1763291784